lib/logger/logger.go

```diff
@@ -46,6 +59,23 @@ func Init() {
 	logAllFlags()
 }
 
+var defaultPrioritySelector = map[string]syslog.Priority{
+	"INFO":  syslog.LOG_INFO,
+	"WARN":  syslog.LOG_WARNING,
+	"ERROR": syslog.LOG_ERR,
+	"FATAL": syslog.LOG_CRIT, //
+	"PANIC": syslog.LOG_EMERG,
+}
+
+func parsePriority(prior string) syslog.Priority {
+	v, found := defaultPrioritySelector[prior]
+	if !found {
+		// default the facility level to LOG_LOCAL7
+		return syslog.LOG_INFO
+	}
+	return v
+}
```

Comment:
Could you move all syslog-related functions to syslog package?
Something like `lib/syslog/logger.go`.